src/views/pages/login/Login.js

```diff
@@ -19,29 +21,45 @@ import { axiosPublic } from 'src/utils/axiosPublic'
 // import { useDispatch, useSelector } from 'react-redux'
 
 const Login = () => {
+  const navigate = useNavigate()
   const [loginLoadingShow, setLoginLoadingShow] = useState(false)
   const [loginErrorMessageShow, setLoginErrorMessageShow] = useState(false)
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
 
-  const toggleAlert = () => {
-    setLoginErrorMessageShow(!loginErrorMessageShow)
-  }
-
   const handleSubmit = async (e) => {
     e.preventDefault()
-    setLoginLoadingShow(!loginLoadingShow)
+    setLoginLoadingShow(true)
     const payload = JSON.stringify({ email, password })
 
     const onSuccess = ({ data }) => {
-      setLoginLoadingShow(!loginLoadingShow)
-      console.log('Logged in successfully')
+      toast.success('Login successful', {
+        position: 'top-right',
+        autoClose: 1200,
+        hideProgressBar: true,
+        closeOnClick: true,
+        pauseOnHover: true,
+        draggable: true,
+        progress: undefined,
+        theme: 'colored',
+      })
+      setLoginLoadingShow(false)
       localStorage.setItem('session', JSON.stringify(data))
-      window.location.reload()
+      navigate('/')
     }
 
     const onFailure = (error) => {
-      toggleAlert()
+      setLoginLoadingShow(false)
+      toast.error(`Login failed ${error && error.response?.data?.detail}`, {
```

Comment:
We want to avoid showing the exact error response that we got from the API. Consider showing **Login failed - wrong credentials supplied** for 401 errors and a blanket **Login failed** for other errors. In the future we shall need to log these failures and ship them to prometheus or some other equivalent tool, I have creates this placeholder issue for the logger #24 .